### ISSUE TITLE
#move default value error

### ISSUE THREAD

**Issue by Alicekb** [learn-co-curriculum/tic-tac-toe-rb#40]
The lab says that the third argument to the move method should have a default value, but the test gives me an error: 
WARNING: Using `expect { }.not_to raise_error(SpecificErrorClass)` risks false positives, since literally any other error would cause the expectation to pass, including those raised by Ruby (e.g. NoMethodError, NameError and ArgumentError), meaning the code you are intending to test may not even get reached. Instead consider using `expect {}.not_to raise_error`. This message can be supressed by setting: `RSpec::Expectations.configuration.warn_about_potential_false_positives = false`. Called from /home/alice/code/labs/tic-tac-toe-rb-q-000/spec/01_tic_tac_toe_spec.rb:55:in `block (3 levels) in <top (required)>'.

When I remove the default value the test passes.

**Comment by ivalentine** [comment]
There seems to be a fundamental inconsistency in this lab. The [`#move` section](https://github.com/learn-co-curriculum/tic-tac-toe-rb#move) of the README says (emphasis mine):

> Your `#move` method must take in three arguments, the board array, the location in the board array that the player would like to fill out with an "X" or and "O", and the player's character (either "X" or "O"). **The third argument, the player's character, should have a default of "X".** Regarding the player's input: if the user's input is `5`, the player wants to fill out position 5 with their character. This means that your method must fill out the correct array index with the player's character. Remember that, from the player's point of view, the board contains spaces 1-9. But, an array's indexes start their count at 0. You'll have to account for that in your `#move` method.

But [this test](https://github.com/learn-co-curriculum/tic-tac-toe-rb/blob/master/spec/01_tic_tac_toe_spec.rb#L46) is testing for `#move` _not_ having a default value for the third argument:

```
describe '#move' do

    it 'does not allow for a default third argument' do
      board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]

      expect{move(board, 1)}.to raise_error(ArgumentError)
    end
```

I've worked with several students so far who have gotten tripped up by this.

**Comment by aturkewi** [comment]
Thanks for pointing this out Ian. It looks like I changed the test, but didn't correctly update the readme to note the change. I'll being updating this in the next day or so. 

I removed the default third arg because there isn't actually a use for it in the full TTT game and it actually leads to more error having it in there.

**Comment by aturkewi** [comment]
Fixed in PR #44
